.github/actions/helm-docs/Dockerfile

```diff
@@ -0,0 +1,8 @@
+
+FROM jnorwood/helm-docs:latest
```

Comment:
I would prefer a fixed version here to prevent changes in generated documentation

Reply:
will change to v1.2.0 👍